lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -624,9 +624,6 @@ def create_module_map(native_target)
             path = target.module_map_path
             UI.message "- Copying module map file to #{UI.path(path)}" do
               contents = custom_module_map.read
-              unless target.requires_frameworks?
-                contents.gsub!(/^(\s*)framework\s+(module[^{}]+){/, '\1\2[system] {')
```

Comment:
can't remove this entirely, as we still need to remove the `framework` specifier